var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -64,6 +63,7 @@ class Neuron(Package):
                         '--without-x']
     _specs_to_options = {
         '+cross-compile': ['cross_compiling=yes',
+                           '--without-readline',
```

Comment:
didn't get why we add here `--without-readline` if we are adding `--with-readline=no` somewhere below.

Reply:
I try to write explanations in the commits :) In cross compile we don't need readline, so we completely disable it. `readline=no` has this special meaning of "use internal"